File: 344-reverse-string/reverse-string.java
class Solution {
    public void reverseString(char[] s) {
        int i = 0;
        int j = s.length - 1;

        while(i < j){
            char s1 = s[i];
            char s2 = s[j];

            s[i] = s2;
            s[j] = s1;

            i++;
            j--;
        }
    }
}

File: 344-reverse-string/reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int i = 0;
        int j = s.size() - 1;

        while(j > i) {
            char temp = s[i];
            s[i] = s[j];
            s[j] = temp;

            i++;
            j--;
        }
        
    }
};